### ISSUE TITLE
Latest bugfix release not on PyPI

### ISSUE THREAD

**Issue by hoechenberger** [psychopy/psychopy#2232]
Hi Jon, the latest release on PyPI is 3.0.0, do you think you could upload 3.0.2 too? Thanks!

**Comment by peircej** [comment]
Yes, sorry. Will do that tomorrow from the office (doesn't seem to allow me via ssh tunnel)
